.github/workflows/gradle.yml

```diff
@@ -68,9 +73,9 @@ jobs:
     - name: Update dev deployment to use latest image
       run: |
         cd config
-        ./kustomize edit set image gcr.io/terra-kernel-k8s/workspace-test:0.0.0=gcr.io/terra-kernel-k8s/workspace-test:${{ github.sha }}
+        ./kustomize edit set image gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:0.0.0=gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:${{ github.sha }}
     - name: Deploy service directly to Dev namepsace
       run: |
-        gcloud container clusters get-credentials terra-kernel-k8s \
-                  --zone us-central1-a --project terra-kernel-k8s
+        gcloud container clusters get-credentials ${GOOGLE_PROJECT} \
```

Comment:
These params are swapped

Reply:
Aw multiline comments didn't work. This should cover ${GOOGLE_PROJECT} and ${GKE_CLUSTER}

Reply:
Good catch!